packages/Popover/src/Popover.js

```diff
@@ -240,9 +269,9 @@ class Popover extends React.Component {
 
     if (this.$tip) {
       tipCoords = getTipCoordinates({
-        tipRect: this.$tip.getBoundingClientRect(),
+        tipRect: this.getBoundingClientRect(this.$tip),
         targetRect,
-        contentRect: this.$content.getBoundingClientRect(),
+        contentRect: this.getBoundingClientRect(this.$content),
         contentCoords,
         align,
       });
```

Comment:
I think this means we need to bump the Popover's version its `package.json`... right @nahumzs?

Reply:
No bump needed
see: https://github.com/acl-services/paprika/pull/110#discussion_r300758329